.travis.yml

```diff
@@ -34,20 +34,19 @@ cache:
 install:
   - bash .travis/maven_cfg.sh
   - bash .travis/start_db.sh
-  - bash .travis/install_utplsql.sh $DB_UT3_USER $DB_UT3_PASS
+  - bash .travis/install_utplsql.sh
   - bash .travis/install_demo_project.sh
-  - bash .travis/install_demo_owner_project.sh
 
 script:
   - echo "Run Unit Tests"
-  - mvn org.jacoco:jacoco-maven-plugin:prepare-agent test
+  - mvn test
   - echo "Run Integration Tests"
-  - mvn org.jacoco:jacoco-maven-plugin:prepare-agent verify -Pintegration -DdbUser="${DB_UT3_USER}" -DdbPass="${DB_UT3_PASS}" -DdbUrl="jdbc:oracle:thin:@${DB_URL}"
```

Comment:
I think this is why we have no coverage on project anymore.
Any reason why `jacoco` was removed?

Reply:
It wasn't removed, it was added in the pom as a build step:
https://github.com/utPLSQL/utPLSQL-maven-plugin/blob/release/3.1.3/pom.xml#L252
https://github.com/utPLSQL/utPLSQL-maven-plugin/blob/release/3.1.3/pom.xml#L400

I remember testing this, I will need to take another look.